host-configs/docker/clang@10.0.0.cmake

```diff
@@ -57,29 +57,29 @@ set(ENABLE_OPENMP ON CACHE BOOL "")
 # TPLs
 #------------------------------------------------------------------------------
 
-set(TPL_ROOT "/home/serac/serac_tpls/clang-10.0.0" CACHE PATH "")
+set(TPL_ROOT "/home/serac/serac_tpls/spack/opt/spack/linux-ubuntu20.04-x86_64/clang-10.0.0" CACHE PATH "")
```

Comment:
This alludes to a missing `config` section in `spack.yaml` for the docker images.

Reply:
@chapman39 can you fix this in a follow up PR?

Reply:
i was wondering about that. yeah i can do that